docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -39,342 +37,123 @@ Devices with Intel® GNA support:
 
```

Comment:
we already have a table with devices here https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md do we need to duplicate them here? I suppose we need to merge this content and update that table with detailed links like here